.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
+
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
```

Comment:
I think we have explicitly stated not to support python 2, so you can remove these.